### PR TITLE
update unity rotate around

### PR DESCRIPTION
**Pull request by valeIT**

### CODE REVIEW

File: _posts/2019-11-04-unity-rotate-around-vs-physics-based-approach.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+published: true
+status: publish
+categories: [Programming]
+image:
+image2:
+author: Valentino Urbano
+---
+
+[This is part 2 of my Unity Space Series. You should first read part 1 if you haven't already][1]
+
```
Comment (MsSlacker) [inline _posts/2019-11-04-unity-rotate-around-vs-physics-based-approach.md:15]: No "."?

File: _posts/2019-11-04-unity-rotate-around-vs-physics-based-approach.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+Recap: I'm playing around with Unity and I wanted to create a cube that would move through space until it got close enough to a planet. When that happened the cube would be attracted by the planet's gravity and start falling on it.
```
Comment (MsSlacker) [inline _posts/2019-11-04-unity-rotate-around-vs-physics-based-approach.md]: Say ship instead of cube, cube seems out of place lol

Comment (MsSlacker) [reply _posts/2019-11-04-unity-rotate-around-vs-physics-based-approach.md]: Into it not on?

File: _posts/2019-11-04-unity-rotate-around-vs-physics-based-approach.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+In Unity there are 2 ways to do it. One using physics and the actual gravitational force of the planet against the gravitational force of the ship, another handling everything manually. We discussed how to implement the second one in the [previous article][1]. After way more testing I can confirm the approach that I took as being the best for my specific use case and wanted to weight the pros and cons of both methods.
```
Comment (MsSlacker) [inline _posts/2019-11-04-unity-rotate-around-vs-physics-based-approach.md:18]: See now you call it a ship! Consistency!

File: _posts/2019-11-04-unity-rotate-around-vs-physics-based-approach.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+## Using Physics
+
+Using Physics the end result is more physically realistic, so if you're going for a space simulation feel this is the route you should take. Hands down. The problem I had was that the force applied was outside of my control. I could control the speed of the ship to keep it in orbit,but if I wanted to so something more custom I could not since gravity would always act on the ship and some ideas we had were not possible to implement with this system given the time constraints.
```
Comment (MsSlacker) [inline _posts/2019-11-04-unity-rotate-around-vs-physics-based-approach.md]: , but - > missing space

Comment (MsSlacker) [reply _posts/2019-11-04-unity-rotate-around-vs-physics-based-approach.md]: if I wanted to so - > do

Comment (MsSlacker) [reply _posts/2019-11-04-unity-rotate-around-vs-physics-based-approach.md]: could not > add comma

Comment (MsSlacker) [reply _posts/2019-11-04-unity-rotate-around-vs-physics-based-approach.md]: We? Who is we?

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2019-11-04-unity-rotate-around-vs-physics-based-approach.md (hunk 1 above)
Comment (valeIT) [reply _posts/2019-11-04-unity-rotate-around-vs-physics-based-approach.md:15]: I think it sounds better with the pause. Maybe a comma?

File: _posts/2019-11-04-unity-rotate-around-vs-physics-based-approach.md (hunk 3 above)
Comment (valeIT) [reply _posts/2019-11-04-unity-rotate-around-vs-physics-based-approach.md:18]: yes...

File: _posts/2019-11-04-unity-rotate-around-vs-physics-based-approach.md (hunk 1 above)
Comment (MsSlacker) [reply _posts/2019-11-04-unity-rotate-around-vs-physics-based-approach.md:15]: Not what I meant...

Comment (valeIT) [reply _posts/2019-11-04-unity-rotate-around-vs-physics-based-approach.md:15]: Makes more sense =)

**Review by MsSlacker — APPROVED** [review]

**Pull request merged**